.github/workflows/lighthouse.yml

```diff
@@ -76,11 +93,11 @@ jobs:
           configPath: './.lighthouserc.json'
           # These URLS capture critical pages / site functionality.
           urls: |
-            ${{ steps.vercel_preview_url.outputs.url }}/en
-            ${{ steps.vercel_preview_url.outputs.url }}/en/about
-            ${{ steps.vercel_preview_url.outputs.url }}/en/about/previous-releases
-            ${{ steps.vercel_preview_url.outputs.url }}/en/download
-            ${{ steps.vercel_preview_url.outputs.url }}/en/blog
+            ${{ needs.get-vercel-preview.outputs.url }}/en
+            ${{ needs.get-vercel-preview.outputs.url }}/en/about
+            ${{ needs.get-vercel-preview.outputs.url }}/en/about/previous-releases
+            ${{ needs.get-vercel-preview.outputs.url }}/en/download
+            ${{ needs.get-vercel-preview.outputs.url }}/en/blog
```

Comment:
need to add learn here

Reply:
this list was semi-intelligently designed not to be exhaustive of all pages, but to try to hit the major features

- homepage (global)
- markdown rendering (about)
- previous-releases (the release data / table)
- download (release data / os detection / lots of complexity)
- blog (blog data / rendering)

Reply:
Lighthouse is not about everything being right, but sampling some of it and measuring performance. Realistically speaking we don't want to sample everything nor people should take it too seriously.

Reply:
thanks @ovflowd - i've had this open all day struggling to find the words. 
if we ever get to that point, we should use playwright to make e2e tests. i suspect chromatic could do more too